docker/consul-template.conf

```diff
@@ -1,10 +1,15 @@
 template {
     source = "/code/docker/settings.py.ctmpl"
     destination = "/code/bot/settings.py"
-    command = "sv hup uwsgi"
 }
 template {
     source = "/code/docker/passwd.ctmpl"
     destination = "/code/passwd"
-    command = "sv hup uwsgi"
+}
+exec {
+    command = "/code/run.py"
```

Comment:
this should have a wrapper script like `lb-run-command` in listenbrainz

Reply:
actually, this could be simpler, by adding a script `run-mb2wdbot` to /usr/local/bin which is:
```
cd /code
exec run.py
```

and set this as the command in the template